### PR TITLE
docs(keyassignment): fix default OpenLinkAtMouseCursor keybind

### PR DESCRIPTION
**Pull request by owittek**
The default keybind to open a link is Shift + Left Click, not CTRL.

### CODE REVIEW

**Comment by owittek** [comment]
Actually it seems more like a bug since I've tried to modify this behavior to use `CTRL` instead of `SHIFT` by adding the snippet from the docs to my config but `SHIFT + Left Click` is still the way to open a link.

I'm running the latest macOS (13.4) in case that helps with testing.

**Comment by bew** [comment]
This section is an example config, not the default.

By default links can be opened directly with a click, or might need shift if the program captures mouse events (shift allows to bypass this capture).

**Comment by owittek** [comment]
> This section is an example config, not the default.
> 
> By default links can be opened directly with a click, or might need shift if the program captures mouse events (shift allows to bypass this capture).

Thank you for your answer. I guess I need to use `Shift` since I'm running everything in tmux so I didn't think about that. Is there some tmux integration to be either able to "just click the link" or to modify this behavior somehow?

Also I'm closing the PR, thank you!

**Comment by bew** [comment]
> or to modify this behavior somehow

I actually asked about it some long time ago at #581, and we now have dedicated fields for making additional custom mouse bindings that apply when the program is capturing mouse events using `mouse_reporting=true`

See https://wezfurlong.org/wezterm/config/mouse.html#configuring-mouse-assignments (below the example)

**Comment by owittek** [comment]
I also managed to get rid of the need to use `Shift` by removing `set -g mouse on` from my `.tmux.conf`. Might be an edge case worth documenting for other people running into this issue.

**Comment by owittek** [comment]
Hey, since I kinda miss being able to scroll I'm once again looking into this. I went into your dotfiles for reference but I saw that you're still referencing the issue as open within them: https://github.com/bew/dotfiles/blob/60fa83e65e56e3550f42312b7d19d9a50be3306a/gui/wezterm/cfg_mouse.lua#L88

Would you mind helping me with a minimal config to enable opening Links with a simple left-click when `set -g mouse on` is set for tmux? @bew

**Comment by bew** [comment]
Yeah I kinda got used to Ctrl-Shift-click as I was on tmux non-stop ^^'

I just updated my config to use `mouse_reporting=true` in my relevant mouse bindings if you want to take a 2nd look:
https://github.com/bew/dotfiles/commit/e37fcbbbe99798bbbacb3dccaa9de2fc79db79e5
(the 1st diff hunk is adding support for additional options to a function that helps me generate mouse bindings, 2nd diff hunk uses that for my Ctrl-click bindings, the 3rd hunk doesn't use my helper function and shows a simple binding to paste with middle-click both in normal output and in a program that enabled mouse reporting)

Basically you need to add new mouse bindings with `mouse_reporting = true` (this binding option is `false` by default, meaning that the binding only works when the program isn't grabbing mouse events)
